netty/src/main/java/io/grpc/transport/netty/GrpcSslContexts.java

```diff
@@ -47,15 +50,30 @@
 public class GrpcSslContexts {
   private GrpcSslContexts() {}
 
-  private static ApplicationProtocolConfig DEFAULT_APN = new ApplicationProtocolConfig(
-          Protocol.ALPN,
-          SelectorFailureBehavior.FATAL_ALERT,
-          SelectedListenerFailureBehavior.FATAL_ALERT,
-          "h2",
-          "h2-17",
-          "h2-16",
-          "h2-15",
-          "h2-14");
+  private static String[] HTTP2_VERSIONS = {
+      "h2",
+      "h2-17",
+      "h2-16",
+      "h2-15",
+      "h2-14"};
+
+  public static ApplicationProtocolConfig ALPN = new ApplicationProtocolConfig(
```

Comment:
We want `NPN_AND_ALPN` to be what people are using for OpenSSL.  To require every client to have to define their own `ApplicationProtocolConfig` for this in order to use OpenSSL seems like a pain point.

Perhaps we change this class to provide `openSsl` versions of the methods?